lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -258,56 +258,43 @@ def add_merchant_description(post, options)
       end
 
       def add_stored_credentials(post, payment, options)
-        return unless stored_credential = options[:stored_credential]
+        return unless options[:stored_credential]
 
-        options = stored_credential_options(stored_credential, options.fetch(:reason_code, ''))
-        post[:processingInformation][:commerceIndicator] = options.fetch(:transaction_type, 'internet')
-        stored_credential[:initial_transaction] ? initial_transaction(post, options) : subsequent_transaction(post, options)
+        post[:processingInformation][:commerceIndicator] = commerce_indicator(options.dig(:stored_credential, :reason_type))
+        add_authorization_options(post, options)
       end
 
-      def stored_credential_options(options, reason_code)
-        transaction_type = options[:reason_type]
-        transaction_type = 'install' if transaction_type == 'installment'
-        initiator = options[:initiator] if  options[:initiator]
-        initiator = 'customer' if initiator == 'cardholder'
-        stored_on_file = options[:reason_type] == 'recurring'
-        options.merge({
-          transaction_type: transaction_type,
-          initiator: initiator,
-          reason_code: reason_code,
-          stored_on_file: stored_on_file
-        })
+      def commerce_indicator(reason_type)
+        case reason_type
+        when 'recurring'
```

Comment:
I'll forward you the latest reply from CyberSource but they basically send that it should only be recurring for non-US transaction if it's a subsequent transaction. If it's an initial transaction it should be recurring_internet for non-US transactions.

I did double check on the API fields PDF for CyberSource Rest and recurring_internet appears as a option for commerceIndicator

Reply:
How can we tell non-US transactions though? Billing address country can come in all sorts of formats

Reply:
Wait I see an option

Reply:
Hmmmm when testing this out in sandbox they **only** want recurring

Reply:
The docs from cybersource rest's email indicates only recurring is desired. I've emailed them for more information